build.js

```diff
@@ -123,6 +124,12 @@ async function build() {
   const swBundle = await rollup.rollup({
     input: "src/lib/sw.js",
     plugins: [
+      // Workbox uses a variable known to be undefined (!) and forces this plugin to be used.
+      // TODO(samthor): This generates statements like "if (1 !== 1)", which are NOT removed from
```

Comment:
Do you have a repro for this? I don't see this issue in our production builds, so I'm wondering what's different between your Rollup builds and ours.

Reply:
So I actually worked out the issue, but it's still fairly confusing.

* Rollup ignores the `if` statements and spits them out regardless.
* Terser on the _command line_ seems to retain the statements (which is what I was testing and was confused by).
* Terser, programatically, deletes them.

It seems like programmatically, Terser has the "conditionals" boolean set, but on the CLI, it does not.

Let me know if that matches your understanding. I can remove or update this comment.